sycamore/functions/document.py

```diff
@@ -30,6 +57,27 @@ def split_and_convert_to_image(doc: Document) -> list[Document]:
 
 
 class DrawBoxes:
+    """
+    DrawBoxes is a class for adding/drawing boxes around elements within images represented as Document objects.
+
+    This class is designed to enhance Document objects representing images with elements (e.g., text boxes, tables)
+    by drawing bounding boxes around each element. It also allows you to customize the color mapping for different
+    element types.
+
+    Args:
+        font_path: The path to the TrueType font file to be used for labeling.
+        default_color: The default color for bounding boxes when the element type is unknown.
+
+    Example:
+
+         .. testcode::
+
+            drawer = DrawBoxes(font_path="path/to/font.ttf", default_color="blue")
+            image_docs =
+                [Document(binary_representation=image_data, properties={"size": (width, height), "mode": "RGB"})]
+            labeled_image_docs = drawer(image_docs)
```

Comment:
Did you generate these examples? It's interesting b/c none of them reference the DocSet, which is how we would expect people to use them. Ideally, I think a better example would be showing how to use this in a map function.

Reply:
Updated